trip_app/conftest.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from accounts.tests.user_factory import faker
+from django.contrib.auth import get_user_model
+
+User = get_user_model()
+
+
+@pytest.fixture
+def signup_data():
+    data = {
+        'username': faker.user_name(),
+        'password': faker.password(),
+        'first_name': faker.first_name(),
+        'last_name': faker.last_name(),
+        'email': faker.email(),
+        'is_staff': False,
+        'is_active': False
+    }
+    return data
+
+
+@pytest.fixture
+def create_user(signup_data, client):
+    """" Create a new user and return user id, username and user password """
```

Comment:
First line of docstring says only about what function does. If you want provide it with information about what returns you can do like this:
```
def my_func(my_param):
    """Do the thing.

    :return: String representing what returns.
    """
```

Reply:
Should be updated now. Thanks.